### ISSUE TITLE
jquery-ui 1.13.0 breaks on minify

### ISSUE THREAD

**Issue by PitWenkin** [leejo/javascript-packer-perl#10]
In the recent versio of [jquery-ui ](https://jqueryui.com/resources/download/jquery-ui-1.13.0.zip
) there are some cases where the minifier breaks the code.

Line 12509 of `jquery-ui.js`

```
options.position = {
  my: "left top",
  at: "left" + ( left >= 0 ? "+" : "" ) + left + " " +
       "top" + ( top >= 0 ? "+" : "" ) + top,
  of: that.window
};
```

is minified to

`options.position={my:"left top",at:"left"+(left>=0?:"")+left+" "+"top"+(top>=0?:"")+top,of:that.window};`

The `"+"`should not be removed at all. Same goes for the `top` part.

There exists a minified version (jquery-ui.min.js) of the same file, which when minified again with this plugin breaks in the same way.

`o.position={my:"left top",at:"left"+(0<=s?"+":"")+s+" top"+(0<=i?"+":"")+i,of:n.window}`

results in

`o.position={my:"left top",at:"left"+(0<=i?:"")+i+" top"+(0<=s?:"")+s,of:n.window}`

As I contact several files which are then minified, unfortunately I can not use the official minified version.

**Comment by leejo** [comment]
Going to have to add a test case to fix this one. Feel free to have a go...

**Comment by PitWenkin** [comment]
The code parts I mentioned or jquery-ui.js in the linked zip are not enough?

**Comment by leejo** [comment]
Yes, they're probably enough. Was just replying to provide an update (and to suggest that you can feel free to have a go as well).

**Comment by PitWenkin** [comment]
As far as my test have gone, the snippet is breaking … but not if I only put that in a file. It seems something above it in the file is responsible …

**Comment by leejo** [comment]
Thanks... that was my worry 😄

**Comment by PitWenkin** [comment]
I made some progress, having selected all `Core` and `Interactions` plugins, and from the widget everything til `Mouse` included, the code will break. When unselecting **only** the `Datepicker` it does however not … 
[Breaking](https://jqueryui.com/download/#!version=1.13.0&themeParams=none&components=111111111111111111111111110000000000000000000000)
[Non-breaking](https://jqueryui.com/download/#!version=1.13.0&themeParams=none&components=111111111111111111111101110000000000000000000000)

**Comment by PitWenkin** [comment]
Seems like some even when removing datepicker, some of the effects are still breaking

**Comment by PitWenkin** [comment]
> As I contact several files which are then minified, unfortunately I can not use the official minified version.

I found a way to resolve my original problem with not being able to use the provided minified versions, therefore I will no longer try to find a solution for this problem.

I will however test any fixes you might propose. Good luck :)

**Comment by leejo** [comment]
Thanks for the input 👍 I'll try to get to this sometime soon.

**Comment by leejo** [comment]
v2.12 on its way to CPAN
